get_github_coderabbit_review/tests/integration/test_github_integration.py

```
@@ -0,0 +1,465 @@
+        self.assertEqual(mock_run.call_count, 2)
+
+    @patch('subprocess.run')
+    def test_json_parsing_error_handling(self, mock_run):
+        """Test handling of malformed JSON responses."""
+        mock_run.return_value = MagicMock(
+            returncode=0,
+            stdout='{"malformed": json',  # Invalid JSON
+            stderr=""
+        )
+
+        with self.assertRaises(Exception):  # Should handle JSON parsing errors
```

Ruff (0.13.1)

Do not assert blind exception: `Exception`

(B017)